docs/kola.md

```diff
@@ -80,6 +81,7 @@ Example format of the file:
 - pattern: test2.test
   ...
 ```
+
```

Comment:
Same here, remove the extra line added here.

Reply:
Thanks renata, finished that there, should be good to go!